### PR TITLE
allow mesos container to run without cmd and args

### PR DESCRIPTION
**Pull request by vixns**
The `docker/runtime` isolator grab cmd or args from docker image if not available in app definition.

### CODE REVIEW

File: src/test/scala/mesosphere/marathon/api/validation/RunSpecValidatorTest.scala [hunk 1]
```diff
@@ -224,6 +224,26 @@ class RunSpecValidatorTest extends MarathonSpec with Matchers with GivenWhenThen
...
+    assert(validate(app).isSuccess)
+    MarathonTestHelper.validateJsonSchema(app)
+  }
+
+  test("mesos container and args") {
+    val f = new Fixture
+    val app = AppDefinition(
+      id = PathId("/test"),
+      args = Some("test" :: Nil),
+      container = Some(f.validMesosDockerContainer))
```
Comment (jdef) [inline src/test/scala/mesosphere/marathon/api/validation/RunSpecValidatorTest.scala]: don't we also need to test validMesosDockerContainer w/o any cmd or args?

**Comment by aquamatthias** [comment]
@vixns Thanks for this PR. Looks good. Can you please rebase this branch?

**Comment by aquamatthias** [comment]
@vixns our test infrastructure is not stable, so I was not able to merge. Can you please rebase?

**Comment by aquamatthias** [comment]
@vixns  This branch has now 29 commits. I rebased your commits on top of master here: #4378

**Pull request closed without merging**